File: src/01-easy/power-of-thor-episode-1/solution.cs
using System;
class Player {
    static void Main() {
        var s = Console.ReadLine().Split(' ');
        int lx = int.Parse(s[0]), ly = int.Parse(s[1]);
        int tx = int.Parse(s[2]), ty = int.Parse(s[3]);
        while (true) {
            Console.ReadLine();
            string dir = "";
            if (ty > ly) { dir += "N"; ty--; }
            else if (ty < ly) { dir += "S"; ty++; }
            if (tx > lx) { dir += "W"; tx--; }
            else if (tx < lx) { dir += "E"; tx++; }
            Console.WriteLine(dir);
        }
    }
}

File: src/01-easy/power-of-thor-episode-1/solution.vb
Module Player
    Sub Main()
        Dim s = Console.ReadLine().Split(" "c)
        Dim lx = CInt(s(0)), ly = CInt(s(1))
        Dim tx = CInt(s(2)), ty = CInt(s(3))
        Do
            Console.ReadLine()
            Dim dir = ""
            If ty > ly Then dir &= "N" : ty -= 1
            ElseIf ty < ly Then dir &= "S" : ty += 1
            End If
            If tx > lx Then dir &= "W" : tx -= 1
            ElseIf tx < lx Then dir &= "E" : tx += 1
            End If
            Console.WriteLine(dir)
        Loop
    End Sub
End Module
